app/routes.py

```diff
@@ -13,6 +16,9 @@
 
 logger = logging.getLogger('app')
 
+cache = Cache()
```

Comment:
This (and the method you write) makes sense, but I think it might make more sense to put it in https://github.com/nein09/polder-federated-search/blob/main/app/__init__.py - this file is more about handling requests that come in to the server and serving responses to them, and `__init__.py` does the setup stuff for the Flask app.